CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.0.0
```

Comment:
I'm not sure if this would be 2.0. Do Django plugins track the main version of Django these days?

Reply:
Sorry, just a suggestion. Was inspired by django-extensions, which I noticed went to v2.0 for Django 2.0 -- django-polymorphic is another example. I'll revert changelog changes for now, your guidance welcome.

Reply:
The "Drop support for Django < 1.8" is still important though. I'll make sure that gets in. I usually generate changelogs off git history.